smart-contracts/contracts/RelayPool.sol

```diff
@@ -418,7 +418,8 @@ contract RelayPool is ERC4626, Ownable {
     // Mark as processed if not
     messages[chainId][bridge][message.nonce] = data;
 
-    uint256 feeAmount = (message.amount * origin.bridgeFee) / 10000;
+    // Calculate fee using fractional basis points (1 = 0.0001 bps)
+    uint256 feeAmount = (message.amount * origin.bridgeFee) / 100000000;
```

Comment:
Let's make a public constant for `100000000` so we can more easily handle things on the front-end and the denomination is clear